examples/A11ySample/src/screens/Advanced/ManagingRefsScreen/ManagingRefsScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback } from "react";
+import { useFocusEffect, useNavigation } from "@react-navigation/native";
+import { StyleSheet, Text } from "react-native";
+import {
+  A11yModule,
+  A11yOrder,
+  useCombinedRef,
+  useDynamicFocusOrder,
+} from "react-native-a11y";
+
+import { DrawerNavigation, DYNAMIC_ORDER } from "../../../../navigation";
+import { NavBar, ReaderExample, Screen } from "../../../components";
+
+export const ManagingRefsScreen = () => {
+  const navigation = useNavigation<DrawerNavigation>();
+  const goBack = () => navigation.navigate(DYNAMIC_ORDER);
+
+  const { a11yOrder, registerOrder, setOrder } = useDynamicFocusOrder();
+  const [firstRef, firstRefCallback] = useCombinedRef<Text>(registerOrder(0));
+  const onLayoutHandler = useCallback(
```

Comment:
I believe this can be moved to a util hook

Reply:
This project is an example, and this code should show how to use our order functionality within other features like setFocus and so on. If you have any ideas on how to make API better I will be really glad to hear. I don't really now what we can improve here